src/app/shared/utilities/util.ts

```diff
@@ -54,4 +54,12 @@ export class Utility {
     }
     return newO;
   }
+
+  mapToObject(map: Map<string, string>): Object {
```

Comment:
why is this needed? does the avro library not understand js map type?

Reply:
Yes for some reason, the AVRO converter can't convert the regular `Map` object, just the json object type.

Reply:
ah ok. Thanks for clarifying.